tests/cluster_file.json

```diff
@@ -0,0 +1,97 @@
+{
+  "_comment": [
+    "Licensed to the Apache Software Foundation (ASF) under one or more",
+    "contributor license agreements.  See the NOTICE file distributed with",
+    "this work for additional information regarding copyright ownership.",
+    "The ASF licenses this file to You under the Apache License, Version 2.0",
+    "(the \"License\"); you may not use this file except in compliance with",
+    "the License.  You may obtain a copy of the License at",
+    "",
+    "http://www.apache.org/licenses/LICENSE-2.0",
+    "",
+    "Unless required by applicable law or agreed to in writing, software",
+    "distributed under the License is distributed on an \"AS IS\" BASIS,",
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.",
+    "See the License for the specific language governing permissions and",
+    "limitations under the License."
+  ],
+  "nodes": [
+    {
+      "hostname": "knode02.knw",
```

Comment:
The format of this changed a bit in 0.6.0 (you need something like `"ssh_config" : { "host": "knode02.knw", "hostname": "", "identityfile": "tests/docker/ssh/id_rsa" }`.

Also, once I tweaked things so I thought they would work I realized that we're not referencing the SSH keys in these settings at all. That must be required for SSH to work, right?